wallet/documentation/docs/libraries/rust/how_to/outputs_and_transactions/_06_create_alias/_code_sample.mdx

```diff
@@ -0,0 +1,19 @@
+import CodeBlock from '@theme/CodeBlock';
+import CodeSample from '!!raw-loader!../../../../../../../examples/create_alias.rs';
+import DotEnvWarning from "../../../../../_admonitions/_warning_dot_env.md"
+
+<DotEnvWarning/>
+
+<CodeBlock className="language-rust">
+    {CodeSample}
+</CodeBlock>
+
+### Run the Example
+
+Before you run the example you should update the `alias_id` to one available in your account.
```

Comment:
Copy paste error? There is nothing with alias_id in this example

Reply:
```suggestion
```